v2/firmware/src/src.ino

```diff
@@ -35,11 +36,12 @@
 enum Effect {
     Colorwheel,
     RGBControl,
+    HSIControl,
     BlueSinus,
 };
 
 // Choose your effect
-static const Effect effect = Colorwheel;
+static const Effect effect = HSIControl;
```

Comment:
Wooaah! You changed the default effect, what an invasive change for our users :wink: